packages/zoe/test/swingsetTests/makeKind/test-makeKind.js

```diff
@@ -1,5 +1,8 @@
 /* global __dirname */
 
+// TODO Remove babel-standalone preinitialization
+// https://github.com/endojs/endo/issues/768
+import '@agoric/babel-standalone';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import '@agoric/install-metering-and-ses';
```

Comment:
is there a version of `prepare-test-env-ava.js` with `metering`? @katelynsills ? @michaelfig ?